test/e2e/storage/csi_inline.go

```diff
@@ -225,4 +229,119 @@ var _ = utils.SIGDescribe("CSIInlineVolumes", func() {
 			framework.Failf("Pod should have been deleted or have DeletionTimestamp, but instead got: %s", retrievedPod)
 		}
 	})
+
+	ginkgo.It("should run through the lifecycle of a CSIDriver", func(ctx context.Context) {
+		// Create client
+		client := f.ClientSet.StorageV1().CSIDrivers()
+		defaultFSGroupPolicy := storagev1.ReadWriteOnceWithFSTypeFSGroupPolicy
+		csiDriverLabel := map[string]string{"e2e-test": f.UniqueName}
+		csiDriverLabelSelector := labels.SelectorFromSet(csiDriverLabel).String()
+
+		// Driver that supports only Ephemeral
+		driver1 := &storagev1.CSIDriver{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "inline-driver-" + string(uuid.NewUUID()),
+				Labels: csiDriverLabel,
+			},
+
+			Spec: storagev1.CSIDriverSpec{
+				VolumeLifecycleModes: []storagev1.VolumeLifecycleMode{storagev1.VolumeLifecycleEphemeral},
+				FSGroupPolicy:        &defaultFSGroupPolicy,
+			},
+		}
+
+		// Driver that supports both Ephemeral and Persistent
+		driver2 := &storagev1.CSIDriver{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "inline-driver-" + string(uuid.NewUUID()),
+				Labels: csiDriverLabel,
+			},
+
+			Spec: storagev1.CSIDriverSpec{
+				VolumeLifecycleModes: []storagev1.VolumeLifecycleMode{
+					storagev1.VolumeLifecyclePersistent,
+					storagev1.VolumeLifecycleEphemeral,
+				},
+				FSGroupPolicy: &defaultFSGroupPolicy,
+			},
+		}
+
+		ginkgo.By("Creating two CSIDrivers")
+		createdDriver1, err := client.Create(ctx, driver1, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
```

Comment:
Can you add some more context/information to all calls of `framework.ExpectNoError` ? It makes it easier to debug when some calls fail and we have to take a look which one of them failed.